api/lambdas/withings/index.js

```diff
@@ -0,0 +1,53 @@
+import axios from "axios";
+
+const getEnvOrFail = name => {
+  const value = process.env[name];
+  if (!value || value.trim().length < 1) throw new Error(`Missing env var ${name}`);
+  return value;
+};
+const api = axios.create();
+const apiServerURL = getEnvOrFail("API_URL");
+
+const buildResponse = (status, body) => ({
+  statusCode: status,
+  body,
+});
+
+const defaultResponse = () => buildResponse(200);
+
+exports.handler = async req => {
+  const withingsWhitelistIpAddresses = [
+    "89.30.121.171",
+    "89.30.121.172",
+    "89.30.121.173",
+    "89.30.121.174",
+    "89.30.121.150",
+    "89.30.121.143",
+    "89.30.121.144",
+    "89.30.121.145",
+    "89.30.121.146",
+    "89.30.121.140",
+    "89.30.121.150",
+    "89.30.121.160",
+    "89.30.121.170",
+  ];
```

Comment:
AFAIK, this Lambda will need to ping Withings to get a list of IPs to use dynamically, as these will change and we can't hardcode them.

Reply:
Also, nit: better to have consts off of the `handler` function as they don't need to be processed on every call.